test/TestDataSet.cpp

```diff
@@ -227,6 +226,34 @@ void TestDataSet::testDataTypeFromString() {
 }
 
 
+void TestDataSet::testDataTypeIsNumeric() {
+    static struct _type_info {
+        bool is_numeric;
+        nix::DataType dtype;
+    } _types[] = {
+            {false, nix::DataType::Bool},
+            {true, nix::DataType::Int8},
+            {true, nix::DataType::UInt8},
+            {true, nix::DataType::Int16},
+            {true, nix::DataType::UInt16},
+            {true, nix::DataType::Int32},
+            {true, nix::DataType::UInt32},
+            {true, nix::DataType::Int64},
+            {true, nix::DataType::UInt64},
+            {true, nix::DataType::Float},
+            {true, nix::DataType::Double},
+            {false, nix::DataType::String},
+            {false, nix::DataType::Nothing}
+    };
```

Comment:
Oh, yeah, I guess so. At least `Opaque`. `Date+DateTime` are never used anywhere. Maybe we should kill them?